pkg/defaults/policies/files/deployment_has_ingress_network_policy.json

```diff
@@ -141,6 +141,42 @@
       ]
     }
   ],
+  "mitreAttackVectors": [
+    {
+      "tactic": "TA0001",
+      "techniques": [
+        "T1133",
+        "T1199"
+      ]
+    },
+    {
+      "tactic": "TA0005",
+      "techniques": [
+        "T1610"
```

Comment:
Not having ingress policy does not indicate that an adversary is deploying container? For example, a system policy to detect the creates of pods would be mapped this technique.